installation/windows.md

```diff
@@ -85,4 +85,103 @@ There is currently no automatic update script for Windows. To update manually, d
 
 ## Starting openHAB as a service
 
-(work in progress...)
+To install the openHAB process as a service in Windows in order to be able to launch it automatically and run it in the background, follow these steps:
+
+1. Complete the [prerequisites](#prerequisites) and regular [installation](#installation) steps, including the package selection
+2. Issue the following 2 commands in your openHAB console:
+```
+feature:install service-wrapper
+wrapper:install --name "openHAB2" --display "openHAB2" --description "openHAB 2 Service"
+```
+![Wrapper Install_Windows](http://imageshack.com/a/img922/1700/teZEiW.jpg)
```

Comment:
No worries, you're using patch-2 in your github fork, so all you'd have to do is go to https://github.com/AngelosF/openhab-docs/tree/patch-2/installation/images and upload you images there. 

You'd then just have to change the URI to "images/uploaded_name.png"